lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -243,6 +265,13 @@ def main():
 
     with tables.open_file(args.input_file, mode='r') as infile:
         image_table = read_table(infile, dl1_images_lstcam_key)
+        # if the image modifier has been used to produce these images, stop here
+        config_from_image_table = json.loads(image_table.meta['config'])
+        if image_modifier_checker(config_from_image_table) and image_modifier_checker(config):
+            log.error(f"The image modifier has already been used to produce the images in file {args.input_file}."
+                        "Re-applying the image modifier is not a good practice, start again from unmodified images please.")
+            sys.exit(0)
```

Comment:
This is an error condition. Exit with something other than 0, e.g. 1

Reply:
thanks for spotting this one